db/ecf_schema.rb

```diff
@@ -207,6 +207,7 @@
     t.datetime "updated_at", null: false
     t.integer "disable_from_year"
     t.boolean "listed", default: false, null: false
+    t.integer "listed_for_school_type_codes", default: [], array: true
```

Comment:
I'm not sure if these changes are intentional; seems unrelated to the PR.

Reply:
I think its ok to keep this this time as we have to keep the ecf schema in sync with ECF